### PR TITLE
Upgrade client dependencies - June 2023

### PR DESCRIPTION
**Pull request by micheal-parks**
This most notably upgrades us from Svelte 3 to 4. I had to hold off on upgrading THREE. There's an unknown issue from r152 -> r153 that disabled vertex colors on point clouds, causing probabilistic gray-scaling rendering of points to fail. I'll investigate this separately.

### CODE REVIEW

File: web/frontend/src/components/slam/2d-renderer.svelte
[the remark this answers is not in the document]
Comment (micheal-parks) [reply web/frontend/src/components/slam/2d-renderer.svelte:1]: Woops, thanks for catching!

**Comment by micheal-parks** [comment]
@ethanlook I had to revert the prime upgrade as well because of `command sh -c npx only-allow pnpm`. We should probably undo this change or ensure that it's isolated to developing in prime - since we shouldn't prevent users from installing prime if they're using npm.

**Comment by ethanlook** [comment]
> command sh -c npx only-allow pnpm

Interesting, my understanding is that this only affects install from prime, not other projects. We definitely don't want to only allow consumers to use pnpm.

**Comment by micheal-parks** [comment]
@Otterverse Would you happen to know why so many checks are failing here? CI is giving issues unrelated to these changes.

**Comment by Otterverse** [comment]
> @Otterverse Would you happen to know why so many checks are failing here? CI is giving issues unrelated to these changes.

I think @edaniels broke it. :-) But also fixed it a short bit ago. https://github.com/viamrobotics/rdk/commit/faf57ab84960fa51849d1087c1b707d741282b3d 

Rebase on main and try again.

**Pull request merged**